src/analytics/analytics-on-redshift.ts

```diff
@@ -393,8 +372,8 @@ function addCfnNag(stack: Stack) {
   addCfnNagToStack(stack, [
 
     ruleRolePolicyWithWildcardResources(
-      'UpsertUsersWorkflow/UpsertUsersStateMachine/Role/DefaultPolicy/Resource',
-      'UpsertUsersWorkflow', 'logs/xray'),
+      'ScanMetadataWorkflow/ScanMetadataStateMachine/Role/DefaultPolicy/Resource',
```

Comment:
Why chaning to ScanMetadata? it's already in mainline

Reply:
removed